### ISSUE TITLE
page scrolled after copying on iphone

### ISSUE THREAD

**Issue by evecalm** [zenorocha/clipboard.js#320]
### Minimal example
To reproduce this bug, just open <https://clipboardjs.com/> on Safari with iOS 10, and tap the last demo button **Copy to clipboard**, then the page should scroll down, but the clipboard is changed.

### Expected behaviour

The web page should stay where it is.

### Actual behaviour

Web page scrolled down. In my own project, the page scrolls up.

### Browsers affected

I'm using Safari with  iPhone 6s, iOS 10.

**Comment by zenorocha** [comment]
Hey @evecalm, thanks for reporting! Just pushed a fix for this, it's still not the best behavior possible but it's better than before. If you have any ideas, please send a pull request.

**Comment by kkateivanova** [comment]
The scroll bug still remains in Safari iOS 10 (iPhone & iPad).

**Comment by zenorocha** [comment]
What version are you using @kkateivanova?

**Comment by kkateivanova** [comment]
@zenorocha the latest - v1.5.15

**Comment by zenorocha** [comment]
Just tested clipboardjs.com on Safari iOS 10.0.2 and everything seems to be working fine.

**Comment by kkateivanova** [comment]
Hey, @zenorocha, I've recorded a video - please, sorry for bad quality https://www.dropbox.com/s/6vv44mln6xyighx/iphone5s_clipboardjs.mov?dl=0.

Here my iPhone 5s, IOS 10.1, safari browser and site clipboardjs.com. There are two problems:
1) When you push the button (without inputs) the screen dragged & scrolled - maybe it happens because keyboard fastly appears/disappears?
2) When you push the button (with inputs) the screen scales to this inputs.
The same problem in iPad ios 10.1

**Comment by a-x-** [comment]
scroll to top issue reproduces on safari and chrome on macos now

clipboard 1.5.15

fix: 
```js
            el.onclick =(e) => e.preventDefault();
            new Clipboard(el);
```

**Comment by jonashaag** [comment]
I can reproduce this, iPhone 4S with latest iOS. It also doesn't copy the text but only scrolls.

Version: 1.5.15

jsfiddle: Here's a jsfiddle with v 1.5.15 and it's still broken https://jsfiddle.net/mkg3x6xk/

**Comment by zenorocha** [comment]
This should be fixed on `v1.6.0`. Let me know if you guys are still having this problem or not.
